src/bootstrap.ts

```diff
@@ -10,31 +10,33 @@ import {
 import { KeplrViewingKeyManager, ViewingKeyManager } from './viewing-keys';
 import { emitEvent } from './events';
 import { getWindow } from './utils';
+import { getFeeForExecute } from './contracts/utils';
 
-const customFees: FeeTable = {
-  upload: {
-    amount: [{ amount: '2000000', denom: 'uscrt' }],
-    gas: '2000000',
-  },
-  init: {
-    amount: [{ amount: '500000', denom: 'uscrt' }],
-    gas: '500000',
-  },
-  exec: {
-    amount: [{ amount: '750000', denom: 'uscrt' }],
-    gas: '750000',
-  },
-  send: {
-    amount: [{ amount: '80000', denom: 'uscrt' }],
-    gas: '80000',
-  },
+const defaultFee: StdFee = {
+  amount: [{ amount: '500000', denom: 'uscrt' }],
+  gas: '500000'
+}
+
+const systemDefaultFees: FeeTable = {
```

Comment:
@David-Sod can you suggest which numbers make more sense?

Reply:
@VPringle97 lets wait for David to adjust those numbers to some that make more sense.